prometheus/README.md

```diff
@@ -0,0 +1,22 @@
+# Indico Prometheus Plugin
+
+This plugin exposes a `/metrics` endpoint which provides Prometheus-compatible output.
+
+![](https://raw.githubusercontent.com/indico/indico-plugins/master/prometheus/screenshot.png)
+
+## prometheus.yml
+```yaml
+scrape_configs:
+  - job_name: indico_stats
+    metrics_path: /metrics
+    scheme: https
+    static_configs:
+      - targets:
+        - yourindicoserver.example.com
```

Comment:
this doesn't need an `https://` prefix?

Reply:
No, that's how the Prometheus config works. It's all about hostnames.